src/api/txMiningAxios.js

```diff
@@ -23,7 +24,13 @@ import axiosWrapperCreateRequestInstance from './axiosWrapper';
  * @param {number} timeout Timeout in milliseconds for the request
  */
 const txMiningRequestClient = (resolve, timeout) => {
-  return axiosWrapperCreateRequestInstance(TX_MINING_URL, resolve, timeout);
+  let txMiningURL;
```

Comment:
Whoever is using the library should be able to set any custom tx mining service url.

Reply:
I agree with you. I think it's a different task from the one in this PR, even though I believe the URL should be customizable. 

I think we can go on with this PR and the other one from the wallet desktop to make it work for both mainnet and testnet only and then start another PR to make it customizable by anyone. What do you think?